### DISCUSSION TITLE
Typo in the docs

### DISCUSSION THREAD

**Discussion by joostvanhoof** [alpinejs/alpine#1720] [4. General]
Sorry not sure how to post improvements to the docs, I'm sure this finds its way.

On [the x-on page](https://alpinejs.dev/directives/on) there's a table with **Modifiers** and its corresponding **Keyboard Key**.

The modifiers for the arrow keys are incorrectly listed:

`up` should be `arrow-up`
`down` should be `arrow-down`
`left` should be `arrow-left`
`right` should be `arrow-right`

Just found this out while I was halfway through writing a bug report 😃

**Comment by joshhanley** [comment]
@joostvanhoof the docs are in this same repo under `Packages>Docs`, so just a PR to that is all thats required :)
